fuzzer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM aflplusplus/aflplusplus
```

Comment:
Is docker really required to make coverage work?

Reply:
It's not meant to be required for coverage to work.

I think it's useful to have for fuzzing though, given the nature of fuzzing:

1. I'd prefer to have the fuzzer somehow containerized, given that I am testing all of this on my local machine and the
[AFLplusplus](https://github.com/AFLplusplus/AFLplusplus) state "you shouldn't be fuzzing on systems where the prospect of data loss is not an acceptable risk". I might not make a difference though, I'm not sure.

2. Since fuzzing is meant to be run for a longer period of time "in production", i.e. probably on a server, I don't want to spend time on portability issues, especially since I am not developing on Linux.

As stated in the README files however, I've at least written a run_test.sh script to test the SWAM server without Docker.

And lastly, the entire docker-compose configuration is set up to work very easily, which I think it currently does. It will work without Docker though, if you install AFLplusplus manually and source the .env file first.

Nonetheless, you may want to check whether you're fine with the Dockerfile I put into the root of this repository.